variables.tf

```diff
@@ -362,4 +362,10 @@ variable "secondary_allow_ip_spoofing" {
   default     = false
 }
 
+variable "existing_kms_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto Services instance in which the key specified in var.kms_key_crn is coming from."
```

Comment:
there is no variable called `kms_key_crn` in this module?

Reply:
Yes, keys are passed in `boot_volume_encryption_key`